docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,433 @@
+umap_transform = umap_transform.fit(vector_df)
+umap_vectors = umap_transform.transform(vector_df)
+umap_df = pd.DataFrame(
+    umap_vectors, columns=["dimension_1", "dimension_2"], index=vector_df.index
+)
+umap_df = umap_df.join(label_df)
+```
+
+Next,we join our dataframes and create a chart, letting us visualize the UMAP-transformed vectors, and coloring them with cluster labels.
+
+```python
+umap_df = umap_df.join(label_df)
+alt.Chart(umap_df).mark_circle(size=8).encode(
+    x="dimension_1", y="dimension_2", color="cluster_label:N"
+).properties(
+    width=600, height=500, title="UMAP Transformed vectors coloured by cluster labels"
+).configure_title(
+    fontSize=16,
+    anchor="middle",
+).configure_legend(
+    strokeColor="black",
+    fillColor="#EEEEEE",
+    padding=10,
+    cornerRadius=10,
+    labelFontSize=14,
+    titleFontSize=14,
+).configure_axis(
+    titleFontSize=14, labelFontSize=12
+)
+```
+
+![cluster visualization](../assets/use_cases/user_acquisition_analytics/cluster_visualization.png)
+
+
+The dark blue clusters (label -1) are outliers - not large or dense enough to form a distinct group. 
+The large number of blobs [better word?] results from the fact that 1/3 of the vector norm mass is made of not very many ad_creatives.
+Note - 2D (UMAP) visualizations often make some clusters look quite dispersed [scattered].
+
+## Understanding the cluster groups
+
+To understand our user clusters better, we can generate some activity histograms.
+
+First, we join user data with cluster labels, create separate DataFrames for each cluster, generates activity histograms for each cluster, and then concatenates these histograms into a single visualization.
+
+```python
+# activity histograms by cluster
+user_df = user_df.set_index("id").join(label_df)
+
+by_cluster_data = {
+    label: user_df[user_df["cluster_label"] == label]
+    for label in np.unique(hdbscan.labels_)
+}
+
+activity_histograms = [
+    alt.Chart(user_df_part)
+    .mark_bar()
+    .encode(x=alt.X("activity", bin=True), y="count()")
+    .properties(title=f"Activity histogram for cluster {label}")
+    for label, user_df_part in by_cluster_data.items()
+]
+
+alt.hconcat(*activity_histograms)
+```
+
+![histograms of clusters](../assets/use_cases/user_acquisition_analytics/cluster_histograms.png)
+
+From our histograms, we can observe that:
+
+- outliers (cluster -1) - a small number of very active users "it means that there are quite a few active users (generally the rates of users turning to active users on a site is quite low), so naturally active users are over-represented outliers"
+- cluster 2 and cluster 3 users are quite similar, but low activity
+- cluster 0 has the highest proportion of medium activity users
+- cluster 1 users are active, "are not outliers and have a fairly balanced activity profile"..."have activities from all over the distribution = meaning here are the active users that are not outliers"
```

Comment:
@morkapronczay
it looks like the vast majority of cluster 1 users have:
- low-moderate activity levels (api calls)
- only very few users very low or very high activity levels
rather than being "balanced" across activity levels..

Reply:
If you look at the initial distribution it is very power-law like - so I would say a few high activity level user means a balanced profile compared to the overall distribution .
low number of inactive users is though a valid feedback - I am creating a PR to update the notebook accordingly. Thanks!